src/qml/SoundPage.qml

```diff
@@ -74,54 +76,35 @@ Item {
                                                                                           volumeControl.volume > "0" ? "ios-sound-indicator-low" : "ios-sound-indicator-off"
     }
 
-    Label {
-        //% "Volume %1%"
-        text: qsTrId("id-sound-percentage").arg(volumeControl.volume)
-        font.pixelSize: Dims.l(6)
-        horizontalAlignment: Text.AlignHCenter
-        verticalAlignment: Text.AlignVCenter
-        wrapMode: Text.Wrap
-        anchors.left: parent.left; anchors.right: parent.right
-        anchors.leftMargin: Dims.w(2); anchors.rightMargin: Dims.w(2)
-        anchors.verticalCenter: parent.verticalCenter
-        anchors.verticalCenterOffset: Dims.h(10)
-    }
+    Column {
+        width: parent.width
+        anchors.top: volumeIcon.bottom
 
-    IconButton {
-        iconName: "ios-remove-circle-outline"
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.horizontalCenterOffset: -Dims.w(15)
-        anchors.bottom: parent.bottom
-        anchors.bottomMargin: Dims.h(10)
-        onClicked: {
-            //Un-mute if muted
-            if (preMuteLevel.value > 0) {
-                // Restore pre mute volume value
-                volumeControl.volume = preMuteLevel.value
-                preMuteLevel.value = 0
-            }
-            var newVal = volumeControl.volume - 10
-            if(newVal < 0) newVal = 0
-            volumeControl.volume = newVal
+        Label {
+            //% "Volume"
+            text: qsTrId("id-sound-percentage")
```

Comment:
In the video this Volume shows up as Volume %1%. I think that's due to not updating the translation files.

Reply:
Translations are now updated using the `$ lupdate-qt5 -no-obsolete src/* i18n/*.h -ts i18n/*.ts` command